core/src/main/groovy/com/muwire/core/files/PersisterFolderService.groovy

```diff
@@ -56,6 +56,11 @@ class PersisterFolderService extends BasePersisterService {
     void onFileHashedEvent(FileHashedEvent hashedEvent) {
         persistFile(hashedEvent.sharedFile)
     }
+
+    void onFileDownloadedEvent(FileDownloadedEvent downloadedEvent) {
+        persistFile(downloadedEvent.downloadedFile)
```

Comment:
only if sharing of downloaded files is enabled, see `FileManager` for the handling of the same event

Reply:
Resolved in https://github.com/zlatinb/muwire/pull/37/commits/a891c83518aa7b02c9b3c1d6befcbf84100cc66e